### PR TITLE
Add cases for Webdav COPY to shared folder

### PR DESCRIPTION
**Pull request by PVince81**
Added test case for copy

### CODE REVIEW

**Comment by PVince81** [comment]
@jvillafanez please review. The matching fixes have been merged into all stable branches, so the tests pass. (just ran it against 8.2)

File: lib/oc-tests/test_sharePermissions.py [hunk 1]
```diff
@@ -341,6 +370,30 @@ def move_in_overwrite(self, expected_success = False):
...
+
+        # upload the test file outside the shared dir first
+        self.oc.put_file(target_file, test_file)
+
+        # then copy that one into the shared dir
+        source_file = target_file
+        target_file = os.path.join(self.shared_dir, source_file)
+        return self._copy(source_file, target_file, expected_success)
+
+    def copy_in_overwrite(self, expected_success = False):
```
Comment (jvillafanez) [inline lib/oc-tests/test_sharePermissions.py:385]: Any reason to have these functions duplicated (copy_in_overwrite and copy_in)? The only difference is the target file.

Comment (jvillafanez) [inline lib/oc-tests/test_sharePermissions.py:385]: ok, found it. Ugly but ok.

**Comment by jvillafanez** [comment]
👍

**Pull request merged**